hack/make-rules/test-cmd-util.sh

```diff
@@ -1279,7 +1312,7 @@ run_kubectl_run_tests() {
   # Pre-Condition: no Job exists
   kube::test::get_object_assert jobs "{{range.items}}{{$id_field}}:{{end}}" ''
   # Command
-  kubectl run pi --generator=job/v1 "--image=$IMAGE_PERL" --restart=OnFailure -- perl -Mbignum=bpi -wle 'print bpi(20)' "${kube_flags[@]}"
+  kubectl run pi --generator=job/v1 "--image=$IMAGE_PERL" --restart=OnFailure -- perl -Mbignum=bpi -wle 'print bpi(20)'
```

Comment:
odd change.  Why did `kube_flags` go away?

Reply:
They are after `--` which means they will be passed to the `perl` command, not to the kube. I don't think they are needed, and if they are they should eventually be before `--`. I'll double check.